lib/Connection.js

```diff
@@ -37,6 +40,20 @@ function Connection(router, req, res) {
             }
         }
 
+        // Cache similar GET requests
```

Comment:
Similar meaning the resource id, props, and other query string parameters are the same. I could improve this by sorting the props and query params, but that's not required for the use case i'm optimizing for (e.g. generating the same symlink in a loop)